src/Microsoft.Crank.JobOjectWrapper/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+
+if (args.Length == 0)
+{
+    Console.WriteLine("Arguments are missing.");
+    Environment.Exit(-1);
+}
+
+Console.WriteLine("Job Object Wrapper");
+Console.WriteLine("Waiting for Job Object to be setup...");
+
+await Task.Delay(1000);
+
+var process = new Process()
```

Comment:
FYI Arcade has this thing called RemoteExecutor which essentially does the same here I think. No meed to change, just in case you want to consider.

Reply:
Is there any description of what RemoteExecutor's goal is exactly. I didn't get the relationship with this after a quick look at the code.